suricata/update/main.py

```diff
@@ -1264,6 +1267,12 @@ def _main():
         logger.info("Loading %s.", drop_conf_filename)
         drop_filters += load_drop_filters(drop_conf_filename)
 
+    # Load user provided output rules filename
+    rule_filename=config.get("output-rule-filename")
+    if rule_filename:
+        logger.info("Setting Rules Output Filename to %s", rule_filename)
```

Comment:
Capitalization of the log message here does not seem to be in accordance with the other log messages.